src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamBroker.cs

```diff
@@ -235,12 +235,20 @@ private bool IsAadOnlyAuthority(Authority authority)
             WebTokenRequest webTokenRequest = await wamPlugin.CreateWebTokenRequestAsync(
                 provider,
                 authenticationRequestParameters,
-                isForceLoginPrompt: false,
+                isForceLoginPrompt: false, 
                 isInteractive: true,
                 isAccountInWam: true)
            .ConfigureAwait(false);
 
-            WamAdapters.AddMsalParamsToRequest(authenticationRequestParameters, webTokenRequest);          
+            // because of https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2476
```

Comment:
this seems like a work around, shouldn't the actual fix be in WAM? maybe create a work time so we come back to this later?

Reply:
I'm keeping the work item open, since this does not fully fix the problem. Discussion with WAM is ongoing.

Reply:
Agreed it's a hack.